exercises/concept/hyperia-forex/HyperiaForexTests.cs

```diff
@@ -7,23 +7,18 @@ public class OperatorOverloadingTests
     [Fact]
     public void Equality_true()
     {
-        Assert.True(Eq(new CurrencyAmount(55, "HD"), new CurrencyAmount(55, "HD")));
+        Assert.True(new CurrencyAmount(55, "HD") == new CurrencyAmount(55, "HD"));
     }
 
     [Fact(Skip = "Remove this Skip property to run this test")]
     public void Equality_false()
     {
-        Assert.False(Eq(new CurrencyAmount(55, "HD"), new CurrencyAmount(60, "HD")));
+        Assert.False(new CurrencyAmount(55, "HD") == new CurrencyAmount(60, "USD"));
```

Comment:
There is a small copy-paste mistake here:


```suggestion
        Assert.False(new CurrencyAmount(55, "HD") == new CurrencyAmount(60, "HD"));
```

Reply:
Sorry, fixed it